web_sheet_full_width_selective/models/model.py

```diff
@@ -0,0 +1,59 @@
+# coding: utf-8
```

Comment:
> It's a question of homogeneity in the files layout, no matter the number of lines. Please split the file and name each file with the model name

@pedrobaeza then you consider you can follow the same practices with the same efficiency on all points, for this very small module and a very large modules as brazilian locale or account or connector. Homogeneity can't be full between large and small module. They have different architect constraints
I'm ok to say **"Developers must split python files to have an adhoc readibility. A good balance must be found with class or code with nearest concept in the same file and avoid long files**"

No metric here, just avoid real excess, trust in developer, give us enjoy develop for oca

Any text editor (often free) have a navigator class plugin (vim, pycharm, komodo, sublime). If they are there it's to be used ?

You talk about one2many relation exception. But they other cases where foreign is narrow between models: think about inheritS. You may have 'res.partner' with inherits model 'res.partner.skilled' or 'res.partner.carrier'. Have them in the same file is really benefit for readibility.

We shouldn't define all exceptions strictly. In which case it can be in the same file or not. Only general approach tendency should be promote. We works an open source project all day. We must keep our minds opened. **Objective is readibility and productivity** of the developer (so many work to do tomorrow).
Rules must help OCA to grow and stay up to date with each odoo version, not bring us to restricted our contributions to oca

I'm not convinced we should follow strict rule on all points. Point on this matter could be simplify.

I know Pedro, you're finally a pragmatic developer, and community leader which have a long term vision. Keep important rules (python mostly). 

I have ever convinced you on an important point previously because you're pragmatically opened.

my 2 cents.

Thanks